### PR TITLE
Create 2023-10-11-introducing-grails-6_1_0.md

### PR DESCRIPTION
**Pull request by puneetbehl**
It highlights the changes and improvements made in this release, provides information about bug fixes, dependency upgrades, and acknowledges the contributions of various developers.

### CODE REVIEW

File: posts/2023-10-11-introducing-grails-6_1_0.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+    3. To migrate from Sitemesh 2 to Sitemesh 3, not only completely decouples Sitemesh from GSP, but it also introduces a powerful new feature that GSP has been lacking: Decorator Chaining. With SiteMesh 3, you can apply multiple decorators to a view. For instance, something like this:
+
+       ````html
+       <meta name="layout" content="my-decorator,my-other-decorator" />
+       ````
+
+       This enhancement provides more flexibility in structuring views and layouts, making it easier to create complex and dynamic page layouts.
+
+    4. Going Further (JDK 17) - Jakarta compatible SiteMesh 3.2 is already available. Sitemesh 3 supports JDK 17, making it an easier transition to future Grails releases, such as Grails 7.0.
```
Comment (mattmoss) [inline posts/2023-10-11-introducing-grails-6_1_0.md:36]: I don't understand this point. Is the Grails SiteMesh3 plugin using SiteMesh 3.2, or is that future?

File: posts/2023-10-11-introducing-grails-6_1_0.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+    5. To learn more about the discussions around Grails and Sitemesh, you can read the details at [here](https://github.com/grails/grails-core/issues/13058). This discussion provides insights into the decisions and considerations made during the decoupling process.
```
Comment (mattmoss) [inline posts/2023-10-11-introducing-grails-6_1_0.md:38]: "…you can read the details at [here](...link...)". I think a better pattern than linking "here" is to say something like "join the discussion" or "join the thread" and link that phrase. So something like:

> To learn more about updates to Grails and Sitemesh, please [review the discussion](https://github.com/grails/grails-core/issues/13058) which provides insights into the decisions and…

But that's personal preference.

**Review by mattmoss — APPROVED** [review]
Overall, looks good. I added a couple review comments, but consider them optional.

File: posts/2023-10-11-introducing-grails-6_1_0.md (hunk 1 above)
Comment (puneetbehl) [reply posts/2023-10-11-introducing-grails-6_1_0.md:36]: Yes, the plugin is using SiteMesh 3.

File: posts/2023-10-11-introducing-grails-6_1_0.md (hunk 2 above)
Comment (puneetbehl) [reply posts/2023-10-11-introducing-grails-6_1_0.md:38]: This make sense, can you please update the PR.

Comment (mattmoss) [reply posts/2023-10-11-introducing-grails-6_1_0.md]: Updated. I also renamed the file prefix to `2023-11-10-` to follow a YYYY-MM-DD format.

**Pull request merged**